util/chunk/mutrow.go

```diff
@@ -346,3 +346,26 @@ func setMutRowJSON(col *column, j json.BinaryJSON) {
 	copy(col.data[1:], j.Value)
 	col.offsets[1] = int32(dataLen)
 }
+
+// ShallowCopyPartialRow shallow copies the data of `row` to MutRow.
+func (mr MutRow) ShallowCopyPartialRow(colIdx int, row Row) {
+	chk := mr.c
```

Comment:
`chk` is only used once, I think we can remove this variable and use `mr.c` directly.